src/overlays/actors/ovl_Bg_Haka_Bombwall/z_bg_haka_bombwall.c

```diff
@@ -56,7 +56,7 @@ static ColliderCylinderInit sCylinderInit = {
     { 80, 80, 0, { 0, 0, 0 } },
 };
 
-static s16 D_80BD64CC[4] = { 24, 15, 10, 5 };
+static s16 rockScale[4] = { 24, 15, 10, 5 };
```

Comment:
Don't forget to prefix static variables with `s`
```suggestion
static s16 sRockScale[] = { 24, 15, 10, 5 };
```